### PR TITLE
sn: Added command to delete users notes.

### PR DESCRIPTION
**Pull request by JakubNvk**
* User can now delete sticky notes adressed to him.

To: @mrshu @pepol

### CODE REVIEW

**Comment by mrshu** [comment]
@JakubNvk thanks, my only question is whether we want to remove all of them or rather just a specific ones.

I believe the latter would be better

**Comment by JakubNvk** [comment]
@mrshu sure, I could add the option to remove specific notes.

File: sticky_notes.py [hunk 1]
```diff
@@ -67,3 +67,12 @@ def list_stickynotes(self, event):
                 self.delay_task(index,
                                 self.sender(format_note(index, note), event))
             return first_note
+
+    @cmd
+    def del_stickynotes(selfself, event):
```
Comment (Adman) [inline sticky_notes.py]: just one self is enough :)

**Comment by Adman** [comment]
@mrshu this way everyone is allowed to delete any stickynotes. Shouldn't it be done like user can delete only messages made by himself?

**Comment by mrshu** [comment]
@Adman yeah, I believe that you are right. In that case though, we need to remember who actually sent it.

Let's do that first.

**Comment by mrshu** [comment]
@JakubNvk since #88 added some more metadata to notes this should be now doable.

@JakubNvk are you up for the challenge? I am assigning this one to you.

**Comment by JakubNvk** [comment]
@mrshu yes, I am!

**Comment by JakubNvk** [comment]
Closing this because I was missing stuff that was added later.

**Pull request closed without merging**